maintainers/team-list.nix

```diff
@@ -44,6 +44,19 @@ with lib.maintainers;
     enableFeatureFreezePing = true;
   };
 
+  android = {
+    members = [
+      numinit
+      hadilq
+      RossComputerGuy
+      # adrian-gierakowski has no entry in maintainer-list.nix
```

Comment:
You only have to add yourself to the maintainers/maintainers-list.nix - committer is a different thing and usually requires longer term work on nixpkgs (note that I don't even currently have it, though I can make a team because I'm already a maintainer). So if you add yourself as a maintainer I can add you to the team in this PR.

Reply:
@adrian-gierakowski Make a PR and link it here, follow the comments and add yourself to `maintainers/maintainers-list.nix`. I can merge it.

Reply:
Thanks! Done: https://github.com/NixOS/nixpkgs/pull/368646

Reply:
@numinit can now add @adrian-gierakowski to the team list. I also gave @numinit an ownership permission over the team so he should be able to add new people too.

Reply:
Done.